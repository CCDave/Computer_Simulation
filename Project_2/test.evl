// a comment
module top;

	wire [15:0] a;

	not(a[1]);
	
endmodule
