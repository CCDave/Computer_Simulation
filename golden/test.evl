// a comment
module top;

	wire [2:0]out,in1,in2;

	and name(out[1],in1[1],in2[1]);
	
endmodule
