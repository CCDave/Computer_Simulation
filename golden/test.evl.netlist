9 1
net in1[0] 0
net in1[1] 1
pin and name 1
net in1[2] 0
net in2[0] 0
net in2[1] 1
pin and name 2
net in2[2] 0
net out[0] 0
net out[1] 1
pin and name 0
net out[2] 0
gate and name 3
pin 1 out[1]
pin 1 in1[1]
pin 1 in2[1]
