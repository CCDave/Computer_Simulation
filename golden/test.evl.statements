STATEMENT 1 MODULE, 3 tokens
STATEMENT 2 WIRE, 12 tokens
STATEMENT 3 COMPONENT, 19 tokens
STATEMENT 4 ENDMODULE
