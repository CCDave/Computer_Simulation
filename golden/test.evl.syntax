module top 3 1
wire in1 3
wire in2 3
wire out 3
component and name 3
pin out 1 1
pin in1 1 1
pin in2 1 1
