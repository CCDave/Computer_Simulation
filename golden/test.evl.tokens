NAME module
NAME top
SINGLE ;
NAME wire
SINGLE [
NUMBER 2
SINGLE :
NUMBER 0
SINGLE ]
NAME out
SINGLE ,
NAME in1
SINGLE ,
NAME in2
SINGLE ;
NAME and
NAME name
SINGLE (
NAME out
SINGLE [
NUMBER 1
SINGLE ]
SINGLE ,
NAME in1
SINGLE [
NUMBER 1
SINGLE ]
SINGLE ,
NAME in2
SINGLE [
NUMBER 1
SINGLE ]
SINGLE )
SINGLE ;
NAME endmodule
