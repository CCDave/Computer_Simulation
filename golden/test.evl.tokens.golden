NAME module
NAME top
SINGLE ;
NAME wire
NAME s0
SINGLE ,
NAME s1
SINGLE ;
NAME dff
SINGLE (
NAME s0
SINGLE ,
NAME s1
SINGLE )
SINGLE ;
NAME not
SINGLE (
NAME s1
SINGLE ,
NAME s0
SINGLE )
SINGLE ;
NAME endmodule
